File: server/Models/Recipe.cs
namespace all_spice_cs.Models;

public class Recipe
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public string Title { get; set; }
  public string Instructions { get; set; }
  public string Img { get; set; }
  public string Category { get; set; }
  public string CreatorId { get; set; }
  public Account Creator { get; set; }
}

File: client/src/models/Recipe.js
import { DatabaseItem } from "./Databaseitems.js"

export class Recipe extends DatabaseItem {
  constructor(data) {
    super(data)
    this.title = data.title
    this.instructions = data.instructions
    this.img = data.img
    this.category = data.category
    this.creator_id = data.creator_id
  }
}